packages/admin/cms-admin/src/blocks/createRichTextBlock.tsx

```diff
@@ -9,12 +9,15 @@ import {
     EntityInstance,
     Modifier,
     RawDraftContentState,
+    RawDraftEntity,
 } from "draft-js";
 import isEqual from "lodash.isequal";
 import * as React from "react";
 import { FormattedMessage } from "react-intl";
 
 import { RichTextBlockData, RichTextBlockInput } from "../blocks.generated";
+import stateToXML from "../common/stateToXML";
```

Comment:
Please use named export wherever possible. Also, abbreviations should be cased camel-case, in this case `stateToXml`